todo_app/app.py

```diff
@@ -1,22 +1,207 @@
+    url = f"https://api.trello.com/1/boards/{id}"
+    querystring = {"key": key, "token": token}
+    response = requests.request("GET", url, params=querystring)
+    response_json = response.json()
+    board = {"id": id, "name": response_json['name'] }    
+    return board
+
+def get_statuses(id):
+    url = f"https://api.trello.com/1/boards/{id}/lists"
+    querystring = {"key": key, "token": token}
+    response = requests.request("GET", url, params=querystring)  
+    data = [] 
+    response_json = response.json()
+    for a in response_json:
+        data.append({'id': a['id'], 'name': a['name']}) 
+    return data
+
+def get_task(id):
+    url = f"https://api.trello.com/1/cards/{id}"
+    querystring = {"key": key, "token": token}
+    response = requests.request("GET", url, params=querystring)    
+    response_json = response.json()    
+    task = {'id': response_json['id'], 'name': response_json['name'], 'status_id': response_json['idList'], 'board_id':response_json['idBoard'] }
+    return task
+
+def delete_task_id(id):
+    url = f"https://api.trello.com/1/cards/{id}"
+    querystring = {"key": key, "token": token}
+    response = requests.request("DELETE", url, params=querystring)
+    return response.text
+
+def create_task(status_id, task_name):
+    url = f"https://api.trello.com/1/cards"
+    querystring = {"name": task_name, "idList": status_id, "key": key, "token": token}
+    response = requests.request("POST", url, params=querystring).json()
+    task_id = response["id"]
+    return task_id
+
+def move_task_status(status_id, task_id):
+    url = f"https://api.trello.com/1/cards/{task_id}"
+    querystring = {"id": task_id, "idList": status_id, "key": key, "token": token}
+    response = requests.request("PUT", url, params=querystring).json()
+    task_id = response["id"]
+    return task_id
+
+def create_status(board_id, status_name):
+    statuses = get_statuses(board_id)
+    exists = False
+    for status in statuses:
+        if (status_name == status['name']):
+            exists = True
+    if (exists == False):
+        url = f"https://api.trello.com/1/boards/{board_id}/lists"
+        querystring = {"name": status_name, "key": key, "token": token}
+        response = requests.request("POST", url, headers=headers, params=querystring).json()
+        status_id = response["id"]
+        return status_id
+
+def create_board(board_name):
+    url = "https://api.trello.com/1/boards/"
+    querystring = {"name": board_name, "key": key, "token": token}
+    response = requests.request("POST", url, params=querystring).json()
+    board_id = response["shortUrl"].split("/")[-1].strip()
+    return board_id
+
+def delete_board_id(id):
+    url = f"https://api.trello.com/1/boards/{id}"
+    querystring = {"key": key, "token": token}
+    response = requests.request("DELETE", url, params=querystring)    
+    return response.text
+
+def delete_status_id(id):
+    url = f"https://api.trello.com/1/boards/{id}/lists/{id}"
```

Comment:
This is not an endpoint that exists. So you should remove this (and the corresponding app route and link in the template)

Even if it did, the first id would presumably be the board id not the list id but you're using the list id for both?